spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -557,6 +557,8 @@ def is_benign_error(self, error):
             "The system cannot find the path specified",
             # UNC paths, see spyder-ide/spyder#23726
             " UNC ",
+            # Matplotlib error, see spyder-ide/spyder#23726
```

Comment:
```suggestion
            # Matplotlib spurious message, see spyder-ide/spyder#24153
```

Improve comment a bit and reference the right issue.